index.html

```diff
@@ -3,7 +3,129 @@
+            data-te-sidenav-link-ref
+          >
+            <span
+              class="mr-4 [&>svg]:h-4 [&>svg]:w-4 [&>svg]:text-gray-400 dark:[&>svg]:text-gray-300"
+            >
+              <i class="fa-solid fa-magnifying-glass fa-xl"></i>
+            </span>
+            <span>Search</span>
+          </a>
+        </li>
+
+        <li class="relative">
+          <a
+            class="flex h-12 cursor-pointer items-center truncate rounded-[5px] px-6 py-4 text-[0.875rem] text-gray-300 outline-none transition duration-300 ease-linear hover:bg-white/10 hover:outline-none focus:bg-white/10 focus:outline-none active:bg-white/10 active:outline-none data-[te-sidenav-state-focus]:outline-none motion-reduce:transition-none"
+            data-te-sidenav-link-ref
+          >
+            <span
+              class="mr-4 [&>svg]:h-4 [&>svg]:w-4 [&>svg]:text-gray-400 dark:[&>svg]:text-gray-300"
+            >
+              <i class="fa-solid fa-lines-leaning fa-xl"></i>
+            </span>
+            <span>Your Library</span>
+          </a>
+        </li>
+        <li
+          class="text-gray-300 tracking-widest text-xs font-light ml-6 mt-4 mb-2"
+        >
+          PLAYLISTS
+        </li>
+        <li class="relative">
+          <a
+            class="flex h-12 cursor-pointer items-center truncate rounded-[5px] px-6 py-4 text-[0.875rem] text-gray-300 outline-none transition duration-300 ease-linear hover:bg-white/10 hover:outline-none focus:bg-white/10 focus:outline-none active:bg-white/10 active:outline-none data-[te-sidenav-state-focus]:outline-none motion-reduce:transition-none"
+            data-te-sidenav-link-ref
+          >
+            <span
+              class="mr-4 [&>svg]:h-4 [&>svg]:w-4 [&>svg]:text-gray-400 dark:[&>svg]:text-gray-300"
+            >
+              <i class="fa-solid fa-square-plus fa-xl"></i>
+            </span>
+            <span>Create Playlists</span>
+          </a>
+        </li>
+        <li class="relative">
+          <a
+            class="flex h-12 cursor-pointer items-center truncate rounded-[5px] px-6 py-4 text-[0.875rem] text-gray-300 outline-none transition duration-300 ease-linear hover:bg-white/10 hover:outline-none focus:bg-white/10 focus:outline-none active:bg-white/10 active:outline-none data-[te-sidenav-state-focus]:outline-none motion-reduce:transition-none"
+            data-te-sidenav-link-ref
+          >
+            <span
+              class="mr-4 [&>svg]:h-4 [&>svg]:w-4 [&>svg]:text-gray-400 dark:[&>svg]:text-gray-300"
+            >
+              <i class="fa-solid fa-heart fa-xl"></i>
+            </span>
+            <span>Liked Songs</span>
+          </a>
+        </li>
+        <section class="flex mt-9 m-5 text-gray-300 text-xs font-light m">
+          <div>
+            <ul class="flex">
+              <li class="mr-3">Legal</li>
+              <li class="mr-3">Privacy Center</li>
+              <li class="">privacy policy</li>
```

Comment:
@Bishakha1589 Please correct this text to match the other points. Use capital case here